src/js/iov/TourController.js

```diff
@@ -0,0 +1,378 @@
+'use strict';
+
+import StreamConfiguration from './StreamConfiguration';
+import Logger from '../utils/logger';
+
+const DEFAULT_TOUR_INTERVAL_DURATION = 10;
+const DEFAULT_TOUR_PRELOAD_DURATION = 9;
+
+export default class TourController {
```

Comment:
@brainthinks This may be a good first candidate for unit tests.

Reply:
I'm fine with that.  I'd rather that be a part of another assignment though...

Reply:
@brainthinks agreed